src/vaishnaviVJun20_arrayExamples/Prime.java

```diff
@@ -0,0 +1,66 @@
+package vaishnaviVJun20_arrayExamples;
+
+public class Prime {
+	
+	int count;
+	int sum;
+	void isPrime(int input) {
+		boolean flag = true;	
+		 for(int index=2;index<=input/2;index++) {	
+		   if(input%index==0) {
+			 flag= false;
+			 System.out.println(input+" is not a prime number");
+			 break;
+		   }
+		}
+		if(flag == true)
+			System.out.println(input +" is a Prime number");
+	}
+	
+	void findPrimeNumbers() {
+		
+		for(int index =2;index<=100;index++) {
+			boolean flag = true;
+		    for(int i=2;i<=index/2;i++) {		    	
+			    if(index%i == 0) {
+			    	flag = false;
+			    	break;
+			    }				    
+		    }
+		    if(flag == true)
+		    	count++;		    
+		 } 
+		 System.out.println("count of prime numbers between 2 to 100 is: " +count);
+	}
+	
+	void findSumOfAllPrimeNumbers() {
+		for(int index =2;index<=100;index++) {
+			boolean flag = true;
+		    for(int i=2;i<=index/2;i++) {		    	
+			    if(index%i == 0) {
+			    	flag = false;
+			    	break;
+			    }				    
+		    }
+		    if(flag == true)
+		    	sum +=index;
+		    			    
+		 } 
+		 System.out.println("count of prime numbers between 2 to 100 is: " +sum);
+		
+	}
+	
+	void findAvgOfAllPrimeNumbers() {
```

Comment:
No need to write extra methods, Just create two instance variable sum and count and perform that operation in isPrime method and use those variable in method like:
void findPrimeNumberInGivenRange(int startNumber, int endNumber) {
		for(int index=startNumber; index<=endNumber; index++) {
			verifyNumberIsPrimeOrNot2(index);
		}
		System.out.println("Total count is: "+count);
		System.out.println("Sum of all Prime number is: "+sum);
		System.out.println("Average of all prime number is: "+sum/count);

Reply:
done suggested changes. Thanks for the logic suggestion.